week-1/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Yep, great stuff!
I might have given a more specific name to the function which created a shouty greeting - but that's a very minor point.